src/translation/translations/en.json

```
@@ -188,7 +188,7 @@
     "columns": {
       "borrowApy": "Borrow APY",
       "liquidity": "Liquidity",
-      "market": "Currency",
+      "market": "Asset",
```

```suggestion
      "asset": "Asset",
```
Lets match the key and value when we can

---

Good idea